pkgs/development/libraries/wxwidgets/wxGTK32.nix

```diff
@@ -102,13 +104,17 @@ stdenv.mkDerivation rec {
     "--with-nanosvg"
     "--disable-rpath"
     "--enable-repro-build"
+    "--enable-webrequest"
     (if compat28 then "--enable-compat28" else "--disable-compat28")
     (if compat30 then "--enable-compat30" else "--disable-compat30")
   ] ++ lib.optional unicode "--enable-unicode"
   ++ lib.optional withMesa "--with-opengl"
   ++ lib.optionals stdenv.hostPlatform.isDarwin [
     "--with-osx_cocoa"
     "--with-libiconv"
+    "--with-urlsession" # for wxWebRequest
+  ] ++ lib.optionals (!stdenv.isDarwin) [
```

Comment:
```suggestion
  ] ++ lib.optionals (!stdenv.hostPlatform.isDarwin) [
```

Reply:
Fixed (squashed) in e5f14ba8f2927c639b0d4fad1094c416a098f784.